dev-support/code-formatter/hadoop_idea_formatter.xml

```diff
@@ -0,0 +1,77 @@
+<!--
+/**
```

Comment:
Do we need /** for the licence here?
https://github.com/apache/hadoop/blob/trunk/pom.xml#L2